taglib/asf/asffile.h

```diff
@@ -134,6 +134,7 @@ namespace TagLib {
       class ContentDescriptionObject;
       class ExtendedContentDescriptionObject;
       class HeaderExtensionObject;
+      class CodecListObject;
```

Comment:
I think we can hide them, and it won't affect the binary compatibility.
However, it will be a huge refactoring and should be another try.